scripts/userstyles.yml

```
@@ -621,7 +621,7 @@ userstyles:
       current-maintainers: [*BlankParticle]
   tuta:
     name: Tuta
-    category: productivity
+    categories: [productivity]
```

Tuta is an email client.
```suggestion
    categories: [email_client, productivity]
```